inspec.yml

```diff
@@ -380,3 +380,97 @@ inputs:
   #type: Boolean
   value: false
 
+# V-73159
+- name: max_retry
+  description: maximum number of password retries
+  type: Numeric
+  value: 3
+
+# V-72315
+- name: firewalld_services
+  description: "Services that firewalld should be configured to allow."
+  type: Array
+  # Examples
+  # 'dhcpv6-client',
+  # 'ssh'
+  value: []
+
+- name: firewalld_hosts_allow
+  description: "Hosts that firewalld should be configured to allow."
+  type: Array
+  value: []
+
+- name: firewalld_hosts_deny
+  description: "Hosts that firewalld should be configured to deny."
+  type: Array
+  value: []
+
+- name: firewalld_ports_allow
+  description: "Ports that firewalld should be configured to allow."
+  type: Array
+  # Examples
+  # '22/tcp',
+  # '4722/tcp'
+  value: []
```

Comment:
Hi, I think we should update the `input` files to use a single style of yaml.

```
- name: firewalld_hosts_deny
  description: "Hosts that firewalld should be configured to deny."
  type: Array
  value:
    - 

- name: firewalld_ports_allow
  description: "Ports that firewalld should be configured to allow."
  type: Array
  # Examples
  #  - '22/tcp',
  # - '4722/tcp'
  value:
    - 
```

Reply:
I do too.  As we discussed I don't think that's the way though.
I created the following YAML:
```
---
emptyArray1: []
emptyArray2:
   -
```

When parsed in Python:
```
>>> import yaml
>>> f=open('/tmp/yamlTest.yml')
>>> y=yaml.load(f)
>>> y
{'emptyArray1': [], 'emptyArray2': [None]}
```
As you can see the first way is actually an empty array. The 2nd is an array with a single 'null/None' value.
I guess there are two ways to write YAML: block and flow.
It seems the 'flow' style uses square and curly brackets rather than line items. I prefer the block style, in general (otherwise, why not just use JSON?).
Flow:
```
---
hash: { key1: value1, key2: value2, key3: value3}
array: [value1, value2, value3]
```
Block (I prefer):
```
---
hash:
  key1: value1
  key2: value2
  key3: value3
array:
  - value1
  - value2
  - value3
```